platform/ios/MGLAnnotationView.h

```diff
@@ -0,0 +1,5 @@
+#import <UIKit/UIKit.h>
```

Comment:
We always import MGLTypes.h and insert the `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` macros. That way you don’t have to think so much about nullability as you fill out the interface in the future.